3_javascript/javascript1/week2/homework2ndweek/fullname.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Flight booking</title>
+  </head>
+  <body>
+    <script>
+      const firstname1 = getFullName("Monika", "Marciniak");
+      const firstname2 = getFullName("John", "Smith", false);
+
+      function getFullName(firstName, surName, isWoman = true) {
+        if (isWoman) {
```

Comment:
if you do not pass a value for firstName or surName, then you will get `undefined`
ex:
const firstname1 = getFullName("Marciniak") 
is Lady Marciniak undefined.
This should be fixed so it should not print `undefined`.